Frame.h

```diff
@@ -0,0 +1,39 @@
+#include<stdlib.h>
+
+#include<iostream>
+
+using namespace std;
+
+bool gameover = 0;
+
+class Frame{
+	public:
+	const int height = 50;
+	const int width = 50;
+	int score = 0;
+		
+	void draw(){
+	     system("clear");
+	     for (int itr = 0; itr < width + 2; itr++){
```

Comment:
why width + 2?

Reply:
I will fix that in next step. I am trying to find a nice way to print the frame. 

I did width + 2 to complete the frame as there is two corner walls are missing , but didn't find right way that time.